packages/block-library/src/navigation/edit/placeholder/placeholder-preview.js

```diff
@@ -6,24 +6,18 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { Icon, search } from '@wordpress/icons';
+import { Icon, navigation } from '@wordpress/icons';
 
 const PlaceholderPreview = ( { isLoading } ) => {
 	return (
-		<ul
+		<div
 			className={ classnames(
 				'wp-block-navigation-placeholder__preview',
-				'wp-block-navigation__container',
 				{ 'is-loading': isLoading }
 			) }
 		>
-			<li className="wp-block-navigation-item">&#8203;</li>
-			<li className="wp-block-navigation-item">&#8203;</li>
-			<li className="wp-block-navigation-item">&#8203;</li>
-			<li className="wp-block-navigation-placeholder__preview-search-icon">
-				<Icon icon={ search } />
-			</li>
-		</ul>
+			<Icon icon={ navigation } />
```

Comment:
The compass icon can't be seen with darker backgrounds, which is something to consider given TT2:
![image](https://user-images.githubusercontent.com/27339341/144612420-a537e3c5-3eea-4bc6-b13b-7c59ed13e542.png)